src/test/java/it/gov/pagopa/reward/notification/service/exports/OrganizationExportsServiceImplTest.java

```diff
@@ -0,0 +1,60 @@
+package it.gov.pagopa.reward.notification.service.exports;
+
+import it.gov.pagopa.reward.notification.dto.controller.RewardExportsDTO;
+import it.gov.pagopa.reward.notification.dto.mapper.RewardOrganizationExports2ExportsDTOMapper;
+import it.gov.pagopa.reward.notification.model.RewardOrganizationExport;
+import it.gov.pagopa.reward.notification.repository.RewardOrganizationExportsRepository;
+import it.gov.pagopa.reward.notification.test.fakers.RewardOrganizationExportsFaker;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.mockito.Mockito;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.data.domain.Page;
+import reactor.core.publisher.Flux;
+
+import java.util.List;
+
+class OrganizationExportsServiceImplTest {
+
+    @Autowired
+    private RewardOrganizationExportsRepository rewardOrganizationExportsRepository;
+    private final RewardOrganizationExports2ExportsDTOMapper rewardOrganizationExports2ExportsDTOMapper = new RewardOrganizationExports2ExportsDTOMapper();
+
+    private OrganizationExportsService organizationExportsService;
+
+    @BeforeEach
+    void init() {
+        organizationExportsService = new OrganizationExportsServiceImpl(rewardOrganizationExportsRepository, rewardOrganizationExports2ExportsDTOMapper);
+    }
+
+    @Test
+    void testFindAllBy() {
+        // Given
+        RewardOrganizationExport rewardOrganizationExportMock = RewardOrganizationExportsFaker.mockInstance(1);
+
+        Mockito.when(rewardOrganizationExportsRepository.findAllBy(Mockito.any(), Mockito.any(), Mockito.any(), Mockito.any())).thenReturn(Flux.just(rewardOrganizationExportMock));
```

Comment:
don't use Mockito.any to verify, but the expected values, in order to verify that the service has successfully passed them